resources/views/community/components/user/progression-status/root.blade.php

```diff
@@ -0,0 +1,88 @@
+<?php
+$widthsPreference = request()->cookie('progression_status_widths_preference');
+$widthMode = $widthsPreference;
+if ($widthMode !== 'equal' && $widthMode !== 'dynamic') {
+    $widthMode = 'equal';
+}
+?>
+
+<h4 class="!leading-none mb-2">Progression Status</h4>
+
+<x-user.progression-status.recalc-cta
+    :totalUserPoints="$userHardcorePoints + $userSoftcorePoints"
+    :totalBeatenSoftcoreCount="$totalBeatenSoftcoreCount"
+    :totalBeatenHardcoreCount="$totalBeatenHardcoreCount"
+/>
+
+<div x-data="{ widthMode: '{{ $widthMode }}' }">
+    <div class="flex flex-col sm:flex-row sm:justify-between w-full mb-2">
+        <x-user.progression-status.legend
+            :totalBeatenHardcoreCount="$totalBeatenHardcoreCount"
+            :totalBeatenSoftcoreCount="$totalBeatenSoftcoreCount"
+            :totalCompletedCount="$totalCompletedCount"
+            :totalMasteredCount="$totalMasteredCount"
+        />
+
+        <div class="hidden sm:flex items-center gap-x-1 select-none cursor-pointer text-xs transition md:active:scale-95">
+            <input
+                id="toggle-row-width-mode-checkbox"
+                type="checkbox"
+                autocomplete="off"
+                @if ($widthMode === 'dynamic') checked @endif
+                @change="newWidthMode = widthMode === 'equal' ? 'dynamic' : 'equal';  widthMode = newWidthMode;  setCookie('progression_status_widths_preference', newWidthMode);"
+                class="cursor-pointer"
+            >
+            <label for="toggle-row-width-mode-checkbox" class="cursor-pointer">Dynamic widths</label>
+        </div>
+    </div>
+
+    @if (count($consoleProgress) > 2)
+        <div class="mb-4">
+            <x-user.progression-status.console-progression-list-item
+                label="Total"
+                :unfinishedCount="$totalUnfinishedCount"
+                :beatenSoftcoreCount="$totalBeatenSoftcoreCount"
+                :beatenHardcoreCount="$totalBeatenHardcoreCount"
+                :completedCount="$totalCompletedCount"
+                :masteredCount="$totalMasteredCount"
+            />
+        </div>
+    @endif
+
+    @if ($topConsole)
+        <div class="mb-1.5">
+            <p class="text-xs">Most Recent</p>
+            <x-user.progression-status.console-progression-list-item
+                :consoleId="$topConsole"
+                :unfinishedCount="$consoleProgress[$topConsole]['unfinishedCount']"
+                :beatenSoftcoreCount="$consoleProgress[$topConsole]['beatenSoftcoreCount']"
+                :beatenHardcoreCount="$consoleProgress[$topConsole]['beatenHardcoreCount']"
+                :completedCount="$consoleProgress[$topConsole]['completedCount']"
+                :masteredCount="$consoleProgress[$topConsole]['masteredCount']"
```

Comment:
Thanks, I forgot to check for this. The component class now makes a few calls to our `isValidConsoleId()` util to weed these out.

Reply:
Problem still occurs if I load up a game for a console I've never touched before (like Uzebox)

Reply:
Yep, I was just able to repro that one. Should now be resolved as of https://github.com/RetroAchievements/RAWeb/pull/1816/commits/f7b9b1a887fac5bae8a7303aac0527f6ec4b3dda.

What was happening was the component recognized there was recent activity on that console, but because there was no progress it had nothing to fallback to for rendering tallies for the rows.

I've set some sane fallback values in this commit. You would now see Uzebox as the latest console with 1 Unfinished game. If you stopped playing before earning an achievement, it would disappear from your table altogether (along with that 1 Unfinished game). This is probably desirable.